hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -507,6 +507,12 @@
       sw_images: ["//sw/device/tests/sim_dv:spi_host_tx_rx_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
     }
+    {
```

Comment:
do you want to consider adding this to the smoke regression?
Mostly because this test is really sensitive to how the agents behave, and you wouldn't want a block level related change to cause a lot of debug pain for this kind of test at top level.

Reply:
Sounds like a good idea to me. I'm not totally sure I found the right location to add it, but we'll see if it got roped into the CI job.